pkg/fiat/client.go

```diff
@@ -0,0 +1,85 @@
+package fiat
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+const fiatUrl = "http://localhost:7003"
```

Comment:
Shouldn't this be `http://spin-fiat.spinnaker:7003`?

Reply:
maybe, I was cheating from the arcade client https://github.com/billiford/go-clouddriver/blob/master/pkg/arcade/client.go#L24

Reply:
Ah yeah, arcade is running in the same pod as go-cd so its default will be `localhost`. The default for fiat will be what I posted above.